amundsen_application/static/js/config/config-types.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 export interface AppConfig {
+  badges: BadgeConfigMap;
```

Comment:
Can we add a section to our documentation that is meant to help people discover features without having ti read code: https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/application_config.md